backend/database/migrations/2019_08_30_200942_add_timing_fields_to_visits_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddTimingFieldsToVisitsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('visits', function (Blueprint $table) {
+            $table->integer('page_load_time')->nullable()->default(450);
+            $table->integer('domain_lookup_time')->nullable()->default(150);
```

Comment:
domain lookup time fronted won't send. The only server can make this request and calculate this information.

Reply:
And it shouldn't have default, and others property as well. Default will break statistic